server/services/prismic.js

```diff
@@ -221,11 +221,13 @@ function createExhibitionPromos(allResults: Object): Array<ExhibitionPromo> {
     return {
```

Comment:
We need to try and refactor this into the common app...

Reply:
Yeah, I thought this too. One for another day?

Reply:
Definitely... just keeping it in the zeitgeist.